client/blocks/plugins-update-manager/schedule-notifications-users.tsx

```diff
@@ -0,0 +1,91 @@
+import { CheckboxControl, SearchControl, Spinner } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { Fragment, useCallback, useEffect, useState } from 'react';
+import { useSiteSlug } from 'calypso/blocks/plugins-update-manager/hooks/use-site-slug';
+import useUsersQuery from 'calypso/data/users/use-users-query';
+
+type Props = {
+	initUsers?: number[];
+	onChange?: ( userIds: number[] ) => void;
+};
+
+type User = {
+	ID: number;
+	name: string;
+};
+
+export const ScheduleNotificationsUsers = ( { initUsers = [], onChange }: Props ) => {
+	const siteSlug = useSiteSlug();
+	const translate = useTranslate();
+
+	const [ userSearchTerm, setUserSearchTerm ] = useState( '' );
+	const [ selectedUsers, setSelectedUsers ] = useState< number[] >( initUsers );
+
+	const {
+		data: userPages,
+		isLoading: isUsersLoading,
+		isFetched: isUsersFetched,
+	} = useUsersQuery( siteSlug, { role: 'administrator' } );
```

Comment:
Is there a way to look for all users with the `update_plugins` cap?

Reply:
Not at the moment, I'll look into extending the `WPCOM_JSON_API_List_Users_Endpoint` endpoint to add this, since we'll need it.

Reply:
PR for the endpoint here: https://github.com/Automattic/jetpack/pull/36893

I'll update this PR to adapt to those changes.